usagov/vote-gov-drupal
# VOTE-2587 Config, templates and po file changes related to string migration

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2587

## Description

This PR has 3 objectives:
1. Replace current hardcoded strings in templates with the dynamic variables
2. Add a new 'Submit button label' field for hardcoded string for search component.
3. Remove translations from the po files.

## Deployment and testing

### Post-deploy steps

1. Pull the latest db from prod to local
2. `lando retune`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/media/add/image and make sure the caption field only accepts simple formatting options
2. visit https://vote-gov.lndo.site/admin/content/block/2 and add a Submit Button Label
3. Test the display of each of these components for migrated content
- Email sign up block
- government banner block
- search component block
- contact identifier block
- partnership block
- registration tool paragraph

## Checklist for the Developer




## Checklist for the Peer Reviewers
